narwhals/_arrow/series.py

```diff
@@ -298,6 +299,17 @@ def std(self, ddof: int = 1) -> int:
 
         return pc.stddev(self._native_series, ddof=ddof)  # type: ignore[no-any-return]
 
+    def skew(self) -> float:
+        values = self._native_series.to_numpy()
+        np = get_numpy()
```

Comment:
could we use `import numpy as np  # ignore-banned-import` here? numpy is currently a required dependency of pyarrow, but I don't know if we can count on that always being the case